src/components/Cauldrons.vue

```diff
@@ -0,0 +1,197 @@
+<template>
+  <!-- Description -->
+  <div class="row">
+    <div>
+      <p class="h6 text-light bg-primary p-3 mt-3 mb-1 rounded">
+        Track your alchemy progress of bubbles and see the costs/bonuses of those upgrades.
+      </p>
+    </div>
+  </div>
+
+  <!-- Cauldron selector -->
+  <ul class="nav nav-pills nav-fill mt-3" role="tablist" :style="{background: colorTo(activeGroup, 'Hex')}">
+    <li v-for="(tab, index) in colors" :key="tab" class="nav-item nav-item-alch">
+      <button
+        class="nav-link text-dark"
+        data-bs-toggle="tab"
+        :data-bs-target="'#' + tab"
+        type="button"
+        role="tab"
+        aria-controls="profile"
+        :aria-selected="tab === Orange"
```

Comment:
```suggestion
        :aria-selected="tab === ' Orange'"
```
It's just missing the quotes here to make `Orange` into a String

Reply:
This has been applied. I was already wondering where the error message about orange came from.